tests/test_create_language.rs

```diff
@@ -35,13 +41,7 @@ fn test_post_language_returns_200() {
         StatusCode::Created,
     );
 
-    assert!(
-        db::language_exists(
-            &connection,
-            &created_language,
-        ),
-        "The language has not been inserted."
```

Comment:
we've lost this error message in the transition

Reply:
Done